IBAnimatable/MaskDesignable.swift

```diff
@@ -40,13 +40,13 @@ public extension MaskDesignable where Self: UIView {
     case .circle:
       maskCircle()
     case .parallelogram(let angle):
-      maskParallelogram(with: angle ?? 60)
+      maskParallelogram(with: angle)
     case .polygon(let sides):
-      maskPolygon(with: sides ?? 6)
+      maskPolygon(with: sides)
     case .star(let points):
-      maskStar(with: points ?? 5)
+      maskStar(with: points )
     case .wave(let direction, let width, let offset):
-      maskWave(with: direction ?? .up, width: width ?? 40.0, offset:offset ?? 0.0)
+      maskWave(with: direction ?? .up, width: width, offset: offset)
```

Comment:
Nope, not needed anymore thank you ;). Strange that the compiler doesn't complain about that.